aptos-move/framework/move-stdlib/tests/vector_tests.move

```diff
@@ -84,7 +104,7 @@ module std::vector_tests {
         assert!(!V::is_empty(&v1), 0);
         assert!(V::length(&v1) == 8, 1);
         let i = 0;
-        while (i < 8) {
+        while (i <8) {
```

Comment:
This does not look correct

Reply:
we will fix it.

Reply:
fixed at https://github.com/movebit/movefmt/commit/d94da4a5e323435fec82a90b4ae06c6d5ed5a0a2